data/plugins/skill/mining/mining.rb

```diff
@@ -39,11 +50,29 @@ def start_mine(pickaxe)
   end
 
   def executeAction
-    skills = mob.skill_set
-    level = skills.get_skill(Skill::MINING).current_level
+    level = mob.skill_set.get_skill(Skill::MINING).current_level
     pickaxe = find_pickaxe
+
+    ores = @ore.id.reject { |ore| ore.level > level}
+    ore_id = if object_id == 2491
```

Comment:
This isn't great, what object is the object with an id of 2491? Also, this would look cleaner if it were a simple conditional without the modifier.